[2024-02-19T18:33:52.976+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v08.create_postgres_table manual__2024-02-19T18:33:51.739778+00:00 [queued]>
[2024-02-19T18:33:52.986+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v08.create_postgres_table manual__2024-02-19T18:33:51.739778+00:00 [queued]>
[2024-02-19T18:33:52.987+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-02-19T18:33:53.004+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2024-02-19 18:33:51.739778+00:00
[2024-02-19T18:33:53.010+0000] {standard_task_runner.py:60} INFO - Started process 1259 to run task
[2024-02-19T18:33:53.014+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v08', 'create_postgres_table', 'manual__2024-02-19T18:33:51.739778+00:00', '--job-id', '469', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpn9w_55m1']
[2024-02-19T18:33:53.015+0000] {standard_task_runner.py:88} INFO - Job 469: Subtask create_postgres_table
[2024-02-19T18:33:53.058+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_with_postgres_operator_v08.create_postgres_table manual__2024-02-19T18:33:51.739778+00:00 [running]> on host 55ac849b3b96
[2024-02-19T18:33:53.131+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Shiv' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v08' AIRFLOW_CTX_TASK_ID='create_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2024-02-19T18:33:51.739778+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-19T18:33:51.739778+00:00'
[2024-02-19T18:33:53.132+0000] {sql.py:276} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS dag_runs (
                dt DATE,
                dag_id VARCHAR,
                PRIMARY KEY (dt, dag_id)
            );
        
[2024-02-19T18:33:53.139+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-02-19T18:33:53.146+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-02-19T18:33:53.150+0000] {sql.py:450} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS dag_runs (
                dt DATE,
                dag_id VARCHAR,
                PRIMARY KEY (dt, dag_id)
            );
        , parameters: None
[2024-02-19T18:33:53.157+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v08, task_id=create_postgres_table, execution_date=20240219T183351, start_date=20240219T183352, end_date=20240219T183353
[2024-02-19T18:33:53.186+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-19T18:33:53.211+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
